CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Adds `"xxxsmall"` size (32px width) to the `Image` component.
 - Adds the `"connect"` section hues to the light and dark color palettes.
 - Adds the `"connect"` color variant to the `Breadcrumbs` component.
+- Adds internal `ChakraProvider` component to be able to conditionally include CSS styles through the `resetCSS` and `disableGlobalStyle` props.
+- Adds a `theme` prop to the `DSProvider` to merge in an additional theme file if necessary; this is to support to the `Header` and `Footer` components in the NYPL Header App.
```

Comment:
Do we have documentation on how to do this?  Or what the theme file would look like?

Reply:
I didn't want to make this too much of a public feature, unless if you think it's okay to do?

Reply:
Good point.  Let's hold off on adding explicit documentation for this.